lightning-tag/data-collection-control.md

```diff
@@ -0,0 +1,99 @@
+# Controlling Data Collection Timing
```

Comment:
This specific example while showing several things is mostly related to collecting consent. I would call this something along the lines of Strict Consent API Implementation - then we could have another page that described just doing getConsent / setConsent that doesn't block execution of page.